test/touchevents.test.tsx

```diff
@@ -7,6 +7,7 @@ import type { SeverityLevel } from '@sentry/types';
 import { TouchEventBoundary } from '../src/js/touchevents';
 
 jest.mock('@sentry/core');
+jest.mock('../src/js/tracing', () => ({}));
```

Comment:
Do we need to add tests for the changes on the touchevents class? or are the current tests enough? the diff is quite big, or is just indentation or something?

Reply:
It was just an indentation change because of the first if. I removed the nesting. Hopefully, someday GH will recognize the indentation changes.

There is only one method call added.
https://github.com/getsentry/sentry-react-native/pull/2835/files#diff-a80d07087a46f7f1f4a0d113eb2a83a6cdb37747528155db8e90f6a35e026e54R233-R236